### ISSUE TITLE
autolayout sets weird layout dimensions if no species/reactions

### ISSUE THREAD

**Issue by luciansmith** [adelhpour/SBMLNetwork#33]
If I have a model with nothing but (say) parameters and ask for an autolayout, it gets confused:

```xml
<layout:dimensions layout:width="-4294967235" layout:height="-4294967235"/>
```

(I haven't actually tried to view/validate the results, but that should presumably also be checked.)

**Comment by adelhpour** [comment]
@luciansmith I think you are right. I need to set a check for it.

**Comment by adelhpour** [comment]
Fixed with PR #35
